client/app/components/project/package-list-item.hbs

```diff
@@ -149,6 +149,58 @@
       {{/if}}
     </LinkTo>
   {{/if}}
+
+  {{#if (or
+    (eq @package.dcpPackagetype (optionset 'package' 'dcpPackagetype' 'code' 'PDEIS'))
+    (eq @package.dcpPackagetype (optionset 'package' 'dcpPackagetype' 'code' 'EIS'))
+  )}}
+    {{!-- TODO: When setting up FEIS, add feis.edit and feis.show as the false condition in route below
+    e.g.
+    (if (eq @package.dcpPackagetype (optionset 'package' 'dcpPackagetype' 'code' 'PDEIS')) 'deis.edit' 'feis.edit') --}}
+    <LinkTo
+      @route={{if (eq @package.statuscode (optionset 'package' 'statuscode' 'code' 'PACKAGE_PREPARATION'))
+        (if (eq @package.dcpPackagetype (optionset 'package' 'dcpPackagetype' 'code' 'PDEIS')) 'deis.edit')
+        (if (eq @package.dcpPackagetype (optionset 'package' 'dcpPackagetype' 'code' 'PDEIS')) 'deis.show')
+      }}
```

Comment:
this could potentially be a little hard to debug if something goes wrong.

LinkTo will eagerly fail if the arguments to `@route` don't map to the setup for that route (# of segments, naming, etc).

One way to split this out a little more is to use `{{#let}}`.